app/settings/page.tsx

```diff
@@ -22,6 +46,69 @@ export default function Settings() {
             Export Acorn Logs
           </Button>
         </section>
+        <section>
+          <h2 className="text-2xl font-semibold mb-4">Browser Tool</h2>
+          {loading ? (
+            <p>Loading...</p>
+          ) : (
+            <>
+              <div className="mt-2 mb-2 flex items-center">
+                <Switch
+                  isSelected={settings.headless}
+                  onValueChange={(isSelected) => {
+                    setSettings((prevState) => ({
+                      ...prevState,
+                      headless: isSelected,
+                    }));
+                  }}
+                >
+                  Headless Mode
+                </Switch>
+                <Tooltip
+                  content="Headless Mode runs the browser without a graphical interface."
+                  placement="right"
+                >
+                  <Chip className="ml-2 font-mono cursor-default">i</Chip>
+                </Tooltip>
+              </div>
+              <div className="mt-2 mb-2 flex items-center">
+                <Switch
+                  isSelected={settings.useDefaultSession}
+                  onValueChange={(isSelected) => {
+                    setSettings((prevState) => ({
+                      ...prevState,
+                      useDefaultSession: isSelected,
+                    }));
+                  }}
+                >
+                  Use Default Session
+                </Switch>
+                <Tooltip
+                  content="Use Default Session will use your existing default Google Chrome session. When this setting is enabled, Chrome must be closed before running the browser tool."
```

Comment:
nit:

<details>

<summary>Right now this goes all the way to the edge of my app and its a little hard to read.</summary>

<img width="1269" alt="Screenshot 2024-09-06 at 9 38 07 AM" src="https://github.com/user-attachments/assets/8d43791c-0be6-4c04-98f8-5c31223a9bdc">

</details>

```suggestion
                <Tooltip
                  classNames={{base: "w-1/2"}}
                  content="Use Default Session will use your existing default Google Chrome session. When this setting is enabled, Chrome must be closed before running the browser tool."
```

<details>

<summary>Could we add that max width so it looks more like this?</summary>

<img width="789" alt="Screenshot 2024-09-06 at 9 43 40 AM" src="https://github.com/user-attachments/assets/3dadc937-19d9-4853-9a0d-2669f6bab9ac">

</details>